architecture/0-References/6-Space_Systems_Engineering.yml

```diff
@@ -0,0 +1,2 @@
+title: Space Systems Engineering 14th Ed. 
+url: ''
```

Comment:
url's empty

Reply:
Since it's a textbook, I am unsure as to what URL to place here.

Reply:
Fair point. I don't suppose there's an online ebook somewhere?